homeassistant/components/alexa/handlers.py

```diff
@@ -232,7 +232,27 @@ async def async_api_turn_off(
 
     service = SERVICE_TURN_OFF
     if entity.domain == cover.DOMAIN:
-        service = cover.SERVICE_CLOSE_COVER
+        open_and_close = cover.CoverEntityFeature.OPEN | cover.CoverEntityFeature.CLOSE
```

Comment:
This can be a constant so we intitialize it once

Reply:
Would like this to be moved to the homeassistant constants module, as a field on `CoverEntityFeature` or as a constant to the top of the file? I honestly feel using it inline here is fine, especially since it makes the code more readable and I would guess that the initialization time of this is neglectible, but I'll move it if you want me to

Reply:
No, we only use a constants in `const.py` if they are reused by multiple modules.